### PR TITLE
Enable controller calls for epsilon greedy

### PR DESCRIPTION
**Pull request by kalantar**
Enable support for epsilon greedy algorithm (set strategy to `epsilon_greedy`.
Addresses issue https://github.com/iter8-tools/iter8-controller/issues/18

### CODE REVIEW

**Comment by fabolive** [comment]
@kalantar: do you need help from @kimikowang to write test cases?

**Comment by kalantar** [comment]
@fabolive I've added two test cases for epsilon greedy. Basic rollforward/backward tests. The others will be the same behavior I think.

**Comment by kalantar** [comment]
@kimikowang is this test flakiness or something more severe: https://travis-ci.com/iter8-tools/iter8-controller/builds/136605540#L598-L600

**Comment by kalantar** [comment]
I guess flakiness since I was able to rerun successfully.

**Comment by kimikowang** [comment]
> I guess flakiness since I was able to rerun successfully.

I think so.

File: test/mock_analytics.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 	"net/http"
 	"net/http/httptest"
 
-	cai "github.com/iter8-tools/iter8-controller/pkg/analytics/checkandincrement"
+	cai "github.com/iter8-tools/iter8-controller/pkg/analytics"
```
Comment (kimikowang) [inline test/mock_analytics.go]: Maybe change the abbreviation? (`cai` refers to `checkandincrement`)

Comment (kalantar) [reply test/mock_analytics.go]: Probably a good idea. I was being lazy.

**Pull request merged**